controllers/patchArticles.controller.js

```diff
@@ -0,0 +1,32 @@
+const updateArticles = require('../models/patchArticles.model')
+const getArticleByID = require('../models/getArticleByID.model')
+
+function patchArticles (req, res, next) {
+    const article_id = req.params.article_id
+    const {inc_votes} = req.body
+    if (inc_votes === undefined || typeof inc_votes !== 'number'){
+        res.status(400).send({message: 'Bad Request'})
+        // return
+    }
+    getArticleByID(article_id).then(article => {
```

Comment:
We don't need to make a separate call to the database here to check if the article exists. As a general rule, always let psql throw an error if it's going to, only if it doesn't could we then look at the result and see if we can implement a custom error handler based on that query result. Only if we can't differentiate from this result between happy and sad path do we then need to consider separate database queries to implement error handling.